### DISCUSSION TITLE
opening files from within bb

### DISCUSSION THREAD

**Discussion by gzqn** [Budibase/budibase#4382] [Help] [answered]
Is it possible to open a folder (ie in windows file explorer) or file (ie open an application and display a specific file) or open a file and display in bb app (I presume this is how attachments display though I haven't got far enough to test this :(,  ) however, being able to open an external file makes updating what would otherwise be attachments more flexible and not subject to IT support for people who otherwise shouldn't have access to the backed).

**Comment by mjashanks** [comment, marked as answer]
There is no way to do this. Budibase does not have access to a local filesystem.

Attachments are currently stored in MinIO, which is part of the Budibase stack. To view a file, you download it an open it natively.
